extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/SqlStatementResourceTest.java

```diff
@@ -929,6 +936,33 @@ public void forbiddenTests()
     );
   }
 
+  @Test
+  public void taskIdNotFoundTests()
```

Comment:
Nit: Doesn't seem like a standard test name.
Typical test names start with `test` and/or mention the behaviour or method they verify.